packages/automerge-repo/test/Repo.test.ts

```diff
@@ -380,6 +376,40 @@ describe("Repo", () => {
       teardown()
     })
 
+    it.only("can broadcast a message without entering into an infinite loop", async () => {
```

Comment:
you left the "only" on here -- notably though the broadcast channel worked fine before with two peers. it was adding a third one that broke things.

Reply:
Noted. I will add another peer to make sure.